backend/app/adapter/sqldb/short_link.go

```diff
@@ -286,6 +286,27 @@ func (s ShortLinkSQL) composeParamList(numParams int) string {
 	return parameterStr
 }
 
+// DeleteShortLink deletes an existing user short link.
+func (s ShortLinkSQL) DeleteShortLink(input entity.ShortLinkInput) error {
+	query := fmt.Sprintf("DELETE FROM %s WHERE %s=$1", table.ShortLink.TableName, table.ShortLink.ColumnAlias)
```

Comment:
```suggestion
	query := fmt.Sprintf(`
	"DELETE FROM %s 
	WHERE %s=$1;"
	`, 
	table.ShortLink.TableName, 
	table.ShortLink.ColumnAlias,
)
```
Usually we format the SQL statements like this